### PR TITLE
spike new gradle task

### PR DESCRIPTION
**Pull request by kaanaktas**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CCD-1131

### Change description ###

Spike for new Gradle task

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -95,6 +95,7 @@ env.BEFTA_RESPONSE_HEADER_CHECK_POLICY="JUST_WARN"
 env.ELASTIC_SEARCH_FTA_ENABLED = "false"
 env.PACT_BROKER_FULL_URL = 'https://pact-broker.platform.hmcts.net'
 env.DEFAULT_COLLECTION_ASSERTION_MODE="UNORDERED"
+env.DATA_SETUP_ENVIRONMENT='master'
```
Comment (MSancaktutar) [inline Jenkinsfile_CNP:98]: please clean this up

File: Jenkinsfile_CNP [hunk 2]
```diff
@@ -120,6 +121,7 @@ withPipeline(type, product, component) {
         }
         env.DEFINITION_STORE_HOST      = "http://ccd-definition-store-api-${environmentOfDependencies}.service.core-compute-${environmentOfDependencies}.internal"
         env.ELASTIC_SEARCH_FTA_ENABLED = "true"
+        env.DATA_SETUP_ENVIRONMENT = environmentOfDependencies
```
Comment (MSancaktutar) [inline Jenkinsfile_CNP:124]: we shouldn't need this kind of an env var. I guess this is just left from previous spike.

File: build.gradle [hunk 3]
```diff
@@ -245,7 +245,8 @@ dependencies {
     }
     compile 'org.jooq:jool-java-8:0.9.14'
 
-    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.0.0'
+    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.0.0-ccd-1223.1'
+    testCompile group: 'com.github.hmcts', name: 'befta-fw', version: '7.0.0-ccd-1131.7'
```
Comment (MSancaktutar) [inline build.gradle:249]: either just a new prerelease that brings a new befta version, or a new befta release to be included as below. not both.

File: src/aat/java/uk/gov/hmcts/ccd/datastore/befta/DataLoader.java [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+public class DataLoader extends DataLoaderToDefinitionStore {
+
+    public DataLoader(String dataSetupEnvironment) {
+        super(dataSetupEnvironment);
+    }
+
+    public static void main(String[] args) {
+        if (args.length == 0) {
+            throw new InvalidParameterException();
```
Comment (MSancaktutar) [inline src/aat/java/uk/gov/hmcts/ccd/datastore/befta/DataLoader.java:15]: let's use another Exception. `java.security.InvalidParameterException` is not relevant.

**Review by MSancaktutar — CHANGES_REQUESTED** [review]
Pls see a few comments.

File: Jenkinsfile_CNP (hunk 1 above)
Comment (kaanaktas) [reply Jenkinsfile_CNP]: done

File: Jenkinsfile_CNP (hunk 2 above)
Comment (kaanaktas) [reply Jenkinsfile_CNP]: removed

File: build.gradle (hunk 3 above)
Comment (kaanaktas) [reply build.gradle]: for now I kept **testCompile group: 'com.github.hmcts', name: 'befta-fw', version: '7.0.0-ccd-1131.7'**

File: src/aat/java/uk/gov/hmcts/ccd/datastore/befta/DataLoader.java (hunk 4 above)
Comment (kaanaktas) [reply src/aat/java/uk/gov/hmcts/ccd/datastore/befta/DataLoader.java]: changed it with IllegalArgumentException

**Pull request closed without merging**